.eslintrc.json

```diff
@@ -188,9 +188,7 @@
     },
     {
       "files": ["*.ts"],
-      "rules": {
-        "ngrx/updater-explicit-return-type": "error"
-      }
+      "extends": ["plugin:ngrx/component-store-strict"]
```

Comment:
this config is added in the latest version.
It includes all component store rules, and configures them to give an error (instead of a warning).